src/Routing/RouteSubscriber.php

```diff
@@ -69,12 +69,15 @@ protected function alterRoutes(RouteCollection $collection) {
         "entity.entity_view_display.{$entity_type_id}.view_mode" => [
           'bundle' => $entity_type_id,
         ],
+        "layout_builder.defaults.$entity_type_id.view" => [
+          'bundle' => $entity_type_id,
+        ],
```

Comment:
@puresyntax71 -> I think it may need `{` and `}` -> `layout_builder.defaults.{$entity_type_id}.view`?

Reply:
But that's not solving the view issue :-(